src/processing/execution.rs

```diff
@@ -24,13 +24,21 @@ pub(crate) struct RunCodeOperation {
     default_colors: Colors,
     block_colors: Colors,
     inner: Rc<RefCell<RunCodeOperationInner>>,
+    state_description: RefCell<&'static str>,
```

Comment:
Minor point, but I wonder if instead of this field, we could derive the description from the process' `ProcessState`? And derive/implement `Display` for `ProcessState` and just use that.
There might be a slight complication because I think the process `handle` is an `Option` whose inner value is removed when the process completes.

Reply:
I added styling now and this becomes more complicated as a `String` / `str` won't do anymore.